### PR TITLE
*: update dockerfile and support cluster.

### PR DESCRIPTION
**Pull request by siddontang**
@ngaut @qiuyesuifeng

### CODE REVIEW

**Comment by qiuyesuifeng** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -11,6 +11,52 @@ docker build -t pingcap/pd .
...
+    -e ETCD_INITIAL_CLUSTER=etcd0=http://192.168.12.50:2380,etcd1=http://192.168.12.51:2380,etcd2=http://192.168.12.52:2380 \
+    -e ETCD_ENDPOINTS=192.168.12.50:2379,192.168.12.51:2379,192.168.12.52:2379 \
+    -e PD_ADVERTISE_ADDR=192.168.12.50:1234 \
+    pingcap/pd
 ```
-docker run -d -p 2379:2379 -p 2380:2380 -p 4001:4001 -p 7001:7001 -p 1234:1234 --name pd pingcap/pd 
-```
\ No newline at end of file
+
+node1
```
Comment (ngaut) [inline README.md]: Is it ok that all of the node have the same address ?

Comment (siddontang) [inline README.md]: yes, you can see the run_cluster.sh which run the cluster in the same machine with different ports.

But in production, we should run in different nodes.

**Comment by ngaut** [comment]
LGTM

**Pull request merged**